recipe/meta.yaml

```
@@ -37,7 +37,7 @@ about:
   license: BSD-3-Clause
   license_family: BSD
   license_file: LICENSE
```

```suggestion
  license_file: LICENSE.txt
```

This needs an update